app/src/main/java/org/literacyapp/service/FaceRecognitionTrainingJobService.java

```diff
@@ -16,6 +16,7 @@ public boolean onStartJob(JobParameters jobParameters) {
         Log.i(getClass().getName(), "onStartJob");
         TrainingHelper trainingHelper = new TrainingHelper(getApplicationContext());
         trainingHelper.extractFeatures();
```

Comment:
Does the `extractFeatures()` method execute asynchronously? Or can we expect that all work is done when the method returns and the `trainClassifier()` starts executing?

```
trainingHelper.extractFeatures();
trainingHelper.trainClassifier();
```